### PR TITLE
[MetaSchedule] Schedule Rule: Auto Inline

### PR DESCRIPTION
**Pull request by MasterJH5574**
This PR adds the Meta-Schedule schedule rule "auto inline".

cc @junrushao1994 @comaniac @Hzfengsy @spectrometerHBH @jcf94 

----------
Co-authored-by: Junru Shao <junrushao1994@gmail.com>
Co-authored-by: Xiyou Zhou <xiyou@octoml.ai>
Co-authored-by: Bohan Hou <32121147+spectrometerHBH@users.noreply.github.com>
Co-authored-by: Siyuan Feng <Hzfengsy@sjtu.edu.cn>
Co-authored-by: Hongyi Jin <3231950289@qq.com>
Co-authored-by: Wuwei Lin <wuwei@apache.org>

### CODE REVIEW

File: python/tvm/meta_schedule/testing/__init__.py [hunk 1]
```diff
@@ -15,5 +15,6 @@
 # specific language governing permissions and limitations
 # under the License.
 """Testing utilities in meta schedule"""
+from . import schedule_rule
```
Comment (junrushao1994) [inline python/tvm/meta_schedule/testing/__init__.py:18]: no need to import it

**Review by junrushao1994 — APPROVED** [review]
Otherwise looks good to me. Please fix the CI issue

File: python/tvm/meta_schedule/testing/__init__.py (hunk 1 above)
Comment (MasterJH5574) [reply python/tvm/meta_schedule/testing/__init__.py]: Thanks for pointing it out!

**Comment by junrushao1994** [comment]
```
E           max(((yy: int32 + ry: int32) - 1), 0)
E           and rhs:
E           ((yy: int32 + ry: int32) - 1)
```

Looks like something is problematic with our arithmetic simplifier

**Pull request merged**